integration_tests/internal/pkgr.yml

```diff
@@ -22,14 +22,13 @@ Suggests: true
 Repos:
   - r_validated: "https://metrumresearchgroup.github.io/r_validated"
   - CRAN: "https://cran.rstudio.com"
-  - local: "path/on/filesystem"
+  - local: "/Users/johncarlos/openspace/rpkgs/gh_dev"
 
-Library: "path/to/installation"
+Library: "test-library"
 
 # any libpaths desired to be set
 Libpaths:
-  - "path/to/some/prev/installed"
-  - "another/path/to/some/prev/installed"
+  - "testsets/simple"
```

Comment:
shouldn't probably need any extra libpaths, this is only if there are existing packages ont he system you want included in the package set

Reply:
So is it safe to remove these from everything?

Reply:
yes - we'll want to test that at some point but overall its somewhat of an antipattern

eg you should really have all your deps in one place, but this is an escape hatch in case you need it